waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java

```diff
@@ -332,4 +318,31 @@ public void waitUntilStarted(int retries, long waitDelay, TimeUnit waitDelayTime
     }
   }
 
+  static String getIPAddress() {
```

Comment:
remove these static methods please we're exposing static fields and relying on side effects. This needs refactoring.